metric_learn/constraints.py

```diff
@@ -46,7 +49,8 @@ def positive_negative_pairs(self, num_constraints, same_length=False,
     return a, b, c, d
 
   def _pairs(self, num_constraints, same_label=True, max_iter=10,
-             random_state=np.random):
+             random_state=None):
+    random_state = check_random_state(random_state)
```

Comment:
It's probably safe to remove this check, as all callers of `_pairs` should have done it already.

Reply:
I agree, done